### PR TITLE
fix : correct error when load_file can't found #6313

### PR DESCRIPTION
**Pull request by jensenojs**
## What type of PR is this?

- [ ] API-change
- [x] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #6313

## What this PR does / why we need it:

### CODE REVIEW

**Comment by jensenojs** [comment]
make it more like this
image

Instead of this 
image

File: pkg/sql/plan/build_insert.go [hunk 1]
```diff
@@ -204,7 +204,11 @@ func buildInsertValues(stmt *tree.Insert, ctx CompilerContext) (p *Plan, err err
 	}, nil
 }
 
-func MakeInsertError(id types.T, col *ColDef, rows []tree.Exprs, colIdx, rowIdx int) error {
+func MakeInsertError(id types.T, col *ColDef, rows []tree.Exprs, colIdx, rowIdx int, err error) error {
+	moer := err.(*moerr.Error)
```
Comment (fengttt) [inline pkg/sql/plan/build_insert.go:208]: Use moerr.IsMoErrCode

Comment (jensenojs) [reply pkg/sql/plan/build_insert.go:208]: ok

**Review by fengttt — CHANGES_REQUESTED** [review]
Please fix error code checking code.

**Review by aunjgr — APPROVED** [review]

**Review by ouyuanning — APPROVED** [review]

**Review by fengttt — APPROVED** [review]

**Pull request merged**